src/bundle/Resources/public/js/scripts/udw/copy_subtree.js

```diff
@@ -0,0 +1,33 @@
+(function () {
+    const btns = document.querySelectorAll('.btn--udw-copy-subtree');
+    const form = document.querySelector('form[name="location_copy_subtree"]');
+    const input = form.querySelector('#location_copy_subtree_new_parent_location');
+    const udwContainer = document.getElementById('react-udw');
+    const token = document.querySelector('meta[name="CSRF-Token"]').content;
+    const siteaccess = document.querySelector('meta[name="SiteAccess"]').content;
+    const closeUDW = () => ReactDOM.unmountComponentAtNode(udwContainer);
+    const onConfirm = (items) => {
+        closeUDW();
+
+        input.value = items[0].id;
+        form.submit();
+    };
+    const canSelectContent = ({item}, callback) => callback(item.ContentInfo.Content.ContentTypeInfo.isContainer);
+    const onCancel = () => closeUDW();
+    const openUDW = (event) => {
+        event.preventDefault();
+
+        ReactDOM.render(React.createElement(eZ.modules.UniversalDiscovery, {
+            onConfirm,
+            onCancel,
+            canSelectContent,
+            confirmLabel: 'Copy to location',
+            title: 'Select location',
+            multiple: false,
+            startingLocationId: parseInt(event.currentTarget.dataset.rootLocation, 10),
+            restInfo: {token, siteaccess}
+        }), udwContainer);
+    };
+
+    btns.forEach(btn => btn.addEventListener('click', openUDW, false));
+})();
```

Comment:
Please pass here `window`, `document`, `React` and use it as we do in other places.